commands/convert.py

```
@@ -0,0 +1,179 @@
+このスクリプトは、アップロードディレクトリにある画像ファイルの名前を変更し、sourceディレクトリに移動する処理を行います。
+画像ファイル名は「image_XX.jpeg」の形式で、sourceディレクトリに既に存在する画像の番号の続きになるように設定されます。
+
+機能:
+1. アップロードディレクトリが存在しない場合は作成
+2. sourceディレクトリが存在しない場合は作成
+3. sourceディレクトリの最後の画像番号を確認
+4. アップロードディレクトリの画像を「image_XX.jpeg」の形式にリネームしてsourceディレクトリに移動
+"""
+
+import os
+import shutil
```

Ruff (0.8.2)

`shutil` imported but unused

Remove unused import: `shutil`

(F401)